File: GEOMETRIA COMPUTACIONAL/C#/TELA_01/1039.cs
using System;
class Program
{
    static void Main()
    {
        while (true)
        {
            try
            {
                string[] input = Console.ReadLine().Split();
                int r1 = int.Parse(input[0]);
                int x1 = int.Parse(input[1]);
                int y1 = int.Parse(input[2]);
                int r2 = int.Parse(input[3]);
                int x2 = int.Parse(input[4]);
                int y2 = int.Parse(input[5]);

                double distancia = Math.Sqrt(Math.Pow(x2 - x1, 2) + Math.Pow(y2 - y1, 2));

                if (r1 >= distancia + r2)
                {
                    Console.WriteLine("RICO");
                }
                else
                {
                    Console.WriteLine("MORTO");
                }
            }
            catch (Exception)
            {
                break;
            }
        }
    }
}

File: GEOMETRIA COMPUTACIONAL/PYTHON/TELA_01/1039.py
import math

while True:
    try:
        r1, x1, y1, r2, x2, y2 = map(int, input().split())
        distancia = math.sqrt((x2 - x1)**2 + (y2 - y1)**2)
        
        if r1 >= distancia + r2:
            print("RICO")
        else:
            print("MORTO")
    except EOFError:
        break